### PR TITLE
add save_to, load_from, evaluation mode, reproduction instruction

### PR DESCRIPTION
**Pull request by V3RGANz**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -44,6 +68,9 @@ def main_train(args):
     optimizer_d = setup_optimizer(discriminator, args.learning_rate, weight_decay=0, args=args)
     optimizer_g = setup_optimizer(generator, args.learning_rate, weight_decay=0, args=args)
 
+    if args.laod_from is not None:
```
Comment (eiofmv) [inline main.py]: typo

**Review by eiofmv — CHANGES_REQUESTED** [review]

**Review by eiofmv — APPROVED** [review]
